### PR TITLE
Include eftec/bladeone in composer.json file

### PR DESCRIPTION
**Pull request by shahbaz-mehar**

### CODE REVIEW

**Review by BushraAsif — COMMENTED** [review]
Changed reviewed

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -109,7 +109,7 @@ jobs:
       - uses: actions/checkout@v1
         with:
           repository: AltaPay/plugin-wordpress
-          ref: 'origin/main'
+          ref: '${{env.TEST_BRANCH_NAME}}'
```
Comment (emicha) [inline .github/workflows/deployment.yml:112]: Is this correct?

**Comment by emicha** [comment]
Missing change log

File: .github/workflows/deployment.yml (hunk 1 above)
Comment (maf-embraceit) [reply .github/workflows/deployment.yml:112]: Yes it is. This variable has the current branch name being merged to run the integration tests from.

**Comment by shahbaz-mehar** [comment]
> Missing change log

@emicha Release notes for version 3.2.9 have already been added and merged but we haven't released it yet. As the current task is related to development to remove the dependency files from the repo and include using composer, that's why I haven't included this.

**Comment by emicha** [comment]
@shahbaz-mehar 
Something like:
```
Add support for WordPress 5.9, WooCommerce 6.1.1
```
Could be a good start.

**Comment by shahbaz-mehar** [comment]
> @shahbaz-mehar Something like:
> 
> ```
> Add support for WordPress 5.9, WooCommerce 6.1.1
> ```
> 
> Could be a good start.

@emicha I have updated the release notes.

**Review by BushraAsif — COMMENTED** [review]
Changes reviewed

**Review by emicha — APPROVED** [review]

**Pull request merged**